data/en/qcas_0020.json

```diff
@@ -0,0 +1,761 @@
+                            }
+                        ]
+                    },
+                    {
+                        "id": "significant-changes-reason-block-1",
+                        "title": "Significant changes",
+                        "type": "Question",
+                        "questions": [{
+                            "id": "significant-changes-reason-question-1",
+                            "title": "Please indicate the reasons for any changes in the total investment for {{ metadata['trad_as_or_ru_name'] }}",
+                            "type": "General",
+                            "answers": [{
+                                "id": "significant-changes-reason-answer",
+                                "mandatory": true,
+                                "type": "Checkbox",
+                                "label": "",
+                                "options": [{
+                                        "label": "Start or end of a long term project",
+                                        "value": "Start or end of a long term project",
+                                        "q_code": "146b"
+                                    },
+                                    {
+                                        "label": "Site changes, e.g. openings, closures, refurbishments or upgrades",
+                                        "value": "Site changes, e.g. openings, closures, refurbishments or upgrades",
+                                        "q_code": "146c"
+                                    },
+                                    {
+                                        "label": "End of accounting period or financial year",
+                                        "value": "End of accounting period or financial year",
+                                        "q_code": "146d"
+                                    },
+                                    {
+                                        "label": "Normal movement for time of year",
+                                        "value": "Normal movement for time of year",
+                                        "q_code": "146e"
+                                    },
+                                    {
+                                        "label": "Change of business structure, merger, or takeover",
+                                        "value": "Change of business structure, merger, or takeover",
+                                        "q_code": "146f"
+                                    },
+                                    {
+                                        "label": "One off or unusual investment",
+                                        "value": "One off or unusual investment",
+                                        "q_code": "146g"
+                                    },
+                                    {
+                                        "label": "Introduction/removal of new legislation/incentive",
```

Comment:
More spaces

Reply:
I swear it got lost in transmission -_-. Will update 👍